src/main/resources/application.conf

```diff
@@ -34,6 +34,9 @@ ergo {
 
     # Number of state snapshot diffs to keep. Defines maximum rollback depth
     keepVersions = 200
+
+    # Optional genesis id block. Defines id of a block that should be applied firstly. Uncomment to use
```

Comment:
Description: "Defines an id of the genesis block. Other genesis blocks will be considered invalid."
Put it to `chain` section and add id of current testnet genesis block to comment